src/modules/Users.pm

```diff
@@ -5937,7 +5937,8 @@ sub ImportUser {
 		"gidNumber"	=> $gid,
 		"homeDirectory"	=> $user->{"homeDirectory"} || $user->{"home"} || $existing{"homeDirectory"} || $home,
 		"type"		=> $type,
-		"modified"	=> "imported"
+		"modified"	=> "imported",
+		"authorized_keys"	=> $user->{"authorized_keys"} || $existing{"authorized_keys"} || []
```

Comment:
indentation is broken

Reply:
Sorry, I realize about it after merging the original code in SP2. I'd like to keep such inconsistency here too, if you don't mind. I don't like it, believe me... but...